apps/files_trashbin/lib/UserMigration/TrashbinMigrator.php

```diff
@@ -176,6 +176,6 @@ public function getDisplayName(): string {
 	 * {@inheritDoc}
 	 */
 	public function getDescription(): string {
-		return $this->l10n->t('Deleted files and folders in the trash bin');
+		return $this->l10n->t('Deleted files and folders in the trash bin (May expire during export if you are low on storage space)');
```

Comment:
The month is probably not meant.

```suggestion
		return $this->l10n->t('Deleted files and folders in the trash bin (may expire during export if you are low on storage space)');
```